### ISSUE TITLE
Issue with target link opening multiple modals (and overlays)

### ISSUE THREAD

**Issue by jerkyb** [Swivelgames/PureModals#2]
Hey there. When clicking a target to open a modal the user is able to fire multiple instances and overlays if they're fast enough in clicking the link.

**Comment by Swivelgames** [comment]
Hey Andrew,

Thanks for the issue. I was able to fix this by modifying the `Modal` class to halt opening new modals until the first one clicked is open. This should solve the issue. I'll be implementing the fix and pushing up a PR shortly. Because B&N requires IE9 support, the version you guys are running will require you to make these edits manually.

Just let me know when you've got time to do a Hangouts call and I'll walk you through the fix.

In addition, the PR should outline the changes I made, so it should be simple enough to implement without any help. But I'm more than happy to help you out with it.

**Comment by jerkyb** [comment]
Yup, I should be able to follow your PR tomorrow. I'll be sure to ping you
if I have any issues. Thanks for looking into it for me!
